### ISSUE TITLE
OSK (on screen keyboard) cannot be disabled

### ISSUE THREAD

**Issue by kylebakerio** [pop-os/shell#1503]
**(1) Issue/Bug Description:**
OSK does not honor the `Settings -> Accessibility -> Typing -> Screen Keyboard` toggle. Even when disabled, it will always pop up.

**(2) Steps to reproduce (if you know):**
Should be clear

**(3) Expected behavior:**
Should be clear

**(4) Distribution (run `cat /etc/os-release`):**
NAME="Pop!_OS"
VERSION="22.04 LTS"

**(5) Gnome Shell version:**
Gnome 42.2

**(6) Pop Shell version (run `apt policy pop-shell` or provide the latest commit if building locally):**

```
pop-shell:
  Installed: 1.1.0~1657300798~22.04~8275681
  Candidate: 1.1.0~1658778861~22.04~4cf9e14
  Version table:
     1.1.0~1658778861~22.04~4cf9e14 1001
       1001 http://apt.pop-os.org/release jammy/main amd64 Packages
       1001 http://apt.pop-os.org/release jammy/main i386 Packages
 *** 1.1.0~1657300798~22.04~8275681 100
        100 /var/lib/dpkg/status
```

**(7) Where was Pop Shell installed from:**
recent fresh install, nothing special

**(8) Monitor Setup (2 x 1080p, 4K, Primary(Horizontal), Secondary(Vertical), etc):**
at this moment, have 2560x1600 @ 100% selected. (natively is 3840x2400, X1 extreme gen 4)
Same issue is well documented, see the block-caribou extension (that is not currently updated/available for 22.04)

**(9) Other Installed/Enabled Extensions:**
Tiling Assistant
Vitals
(but both are irrelevant to this issue)

**(10) Other Notes:**
Same issue on girlfriend's Yoga 370, and every other pop machine.

**Comment by kylebakerio** [comment]
The solution from @EngineerDaryl in the [comments on the gnome-extension page](https://extensions.gnome.org/extension/1326/block-caribou/) are working as a temporary hard-disable extension. Annoying, but so be it.

tldr for how to implement it, summarizing steps from [here](https://www.pragmaticlinux.com/2021/06/manually-install-a-gnome-shell-extension-from-a-zip-file/):

1. Select "Download" and get the zip
2. Unzip it
3. Rename the folder you extract to: block-caribou-36@lxylxy123456.ercli.dev
4. Open the metadata.json file in that folder, and delete everything in it, and copy/paste this into that file:

```json
{
"_generated": "Generated by SweetTooth, do not edit",
"description": "Blocks caribou (the on screen keyboard) from popping up when you use a touchscreen. Even if it's disabled in the accessibility services menu. Continuation of keringar's work. Tested on GNOME Shell version 3.36 - 41 on Fedora 32 - 35. For a higher version see https://github.com/lxylxy123456/cariboublocker#installing-on-high-gnome-shell-version .",
"name": "Block Caribou 36",
"shell-version": [
"40",
"41",
"42"
],
"url": "https://github.com/lxylxy123456/cariboublocker",
"uuid": "block-caribou-36@lxylxy123456.ercli.dev",
"version": 4
}
```

5. Copy/paste the folder into .local/share/gnome-shell/extensions
6. press `alt+f2` and run `r` to reload gnome
7. Type "Extensions" into the launcher to open your gnome-shell extensions manager, and enable the block-caribou extensions

**Comment by SourLemonJuice** [comment]
VERSION="22.04 LTS"
Gnome version = 43.9

Bug still exists, that already 22 month.
